src/components/UserProfile.jsx

```diff
@@ -35,6 +38,27 @@ export default function UserProfile({
   const metadataSchemas = useUserMetadataSchemas(userId);
   const { data: agsData, loading: agsLoading } =
     useGetUserUnprocessedAssetGroupSightings(userId);
+  const {
+    data: currentUserData,
+    loading: currentUserDataLoading,
+    error: currentUserDataError,
+  } = useGetMe();
+  const isUserManager = get(
+    currentUserData,
+    'is_user_manager',
+    false,
+  );
+  const {
+    data: nonSelfUserData,
+    loading: nonSelfUserDataLoading,
+    error: nonSelfUserDataError,
+  } = useGetUser(userId);
+
+  const nonSelfCollabData = get(
+    nonSelfUserData,
+    ['collaborations'],
+    [],
+  ); // deleteMe this seems like too small a calculation to memoize
```

Comment:
I agree that it doesn't need to be memoized. I think you can delete the comment.

Reply:
Done